app/views/matches/edit.html.slim

```diff
@@ -1,5 +1,7 @@
 h1.title
   | EDIT MATCH
+  - if not @error.empty?
```

Comment:
generally discourage using `not`
http://stackoverflow.com/questions/27429711/can-someone-explain-not-vs-in-ruby